docs/framework/guides/tutorials/implementing-an-inline-widget.md

```diff
@@ -715,7 +715,7 @@ export default class PlaceholderUI extends Plugin {
 		const placeholderNames = editor.config.get( 'placeholderConfig.types' );            // CHANGED
 
 		editor.ui.componentFactory.add( 'placeholder', locale => {
-			// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.